paperwork_backend/shell.py

```diff
@@ -3,20 +3,21 @@
 import json
 import os
 import sys
+import multiprocessing
+from functools import partial
 
 import gi
 import pyocr
 
-gi.require_version('Gdk', '3.0')
-gi.require_version('PangoCairo', '1.0')
-gi.require_version('Poppler', '0.18')
-
 from . import config
 from . import docimport
 from . import docsearch
 from .labels import Label
 from . import fs
 
+gi.require_version('Gdk', '3.0')
```

Comment:
Please move back those lines before the local imports. There were there for a reason:
We must call `gi.require_version()` before importing any gi module (Cairo, Poppler, etc) (otherwise GLib will complain at runtime). But afaik, we can only call them once in the program (otherwise Glib will complain).
So they are placed at each entry points (see `paperwork/setup.py` and `paperwork-backend/setup.py`) and must be placed before any local import (because each of them may import Cairo, Poppler, etc)

Reply:
Oh sorry. Pep8 has shown this as an error and I missed this.
I will revert this change.